<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тригонометрические выражения</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 228, 255);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string> Введите значение угла в градусах:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.585, fx:0.5, fy:0.5, stop:0 rgba(204, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>Синус</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.585, fx:0.5, fy:0.5, stop:0 rgba(204, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>Косинус</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_4">
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.585, fx:0.5, fy:0.5, stop:0 rgba(204, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 75 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Тангенс</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="styleSheet">
      <string notr="true">
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.585, fx:0.5, fy:0.5, stop:0 rgba(204, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 75 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Котангенс</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton_5">
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.585, fx:0.5, fy:0.5, stop:0 rgba(204, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 75 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Арксинус</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton_6">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.585, fx:0.5, fy:0.5, stop:0 rgba(204, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>Арккосинус</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushButton_8">
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.585, fx:0.5, fy:0.5, stop:0 rgba(204, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 75 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Арктангенс</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButton_7">
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.585, fx:0.5, fy:0.5, stop:0 rgba(204, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 75 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Арккотангенс</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lbl">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Результат в радианах:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="pushButton_9">
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
